### ISSUE TITLE
Foundation Model

### ISSUE THREAD

**Issue by lss11005** [Spico197/Humback#8]
Can I use other model to replace [meta-llama/Llama-2-7b-hf], such as Mistral-7b?
which part should to be modified if I can replace, data preprocess or other config~~

**Comment by Spico197** [comment]
Sure, you can use other models by setting `--model_name_or_path` in `train_seed.sh` and `train_backward_Myx.sh`.

**Comment by lss11005** [comment]
Should other training parameters need to be modified? such as learning_rate, beta, training data~~
Have you tried other models?
I finetuned model(zephyr-7b-sft-full) with seed.jsonl, but test acc drop a lot( test on Winogrande )
zephyr-7b-sft-full         0.7419
m0                                0.5667

**Comment by Spico197** [comment]
No I didn't try on other models. Since you are finetuning on an SFT model, you may want to change the conversation template accordingly. e.g. https://github.com/Spico197/Humback/blob/641153764e59bd4443a2c42b893c9fec6db24467/src/data.py#L17

**Comment by lss11005** [comment]
template  is necessary? can I construct data myself, such as "instruction" -- "response“； and reverse while train M_yx

**Comment by Spico197** [comment]
Sure you can construct dataset yourself. Just need to change the corresponding data construction process accordingly.
